cmd/autoconfigbrancher/main.go

```diff
@@ -50,6 +51,8 @@ func parseOptions() options {
 	fs.StringVar(&o.gitEmail, "git-email", "", "The email to use on the git commit. Requires --git-name. If not specified, uses the system default.")
 	fs.StringVar(&o.targetDir, "target-dir", "", "The directory containing the target repo.")
 	fs.StringVar(&o.assign, "assign", githubTeam, "The github username or group name to assign the created pull request to.")
+	fs.StringVar(&o.whitelist, "whitelist", "", "The path of the whitelisted repositories file.")
```

Comment:
Please make this `--whitelist-file` for consistency.

Reply:
addressed